text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+  * Now hiring (agencies & companies with this field toggled)
+
+#### Person / agency / company page
+
+One template with multiple URL patterns, depending on the Creator type:
+  * /companies/[company-name]
+  * /agencies/[agency-name]
+  * /people/[person-name]
+
+
+### Data Structures
+
+Showcase sites ([listed here](https://github.com/gatsbyjs/gatsby/blob/master/docs/sites.yml)) can have multiple Creators.
+
+**Creators** can either be an Individual, a Digital Agency, or a Company. . 
+
+Creator has the following fields:
+* an id, in slug format (eg mike-allanson)
+* type -- Individual, a Digital Agency, or a Company. We use the word Organization below to refer to either a Digital Agency and Company.
+* a description of them in the third person “Eg, Mike Allanson is a developer living in London"
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
+* People they are affiliated with (if Org)
+* A list of open jobs (if they are Organization type)
+* A list of all the sites they’ve built.
+* A location with:
+    * A description (“London, UK”)
+    * Lat / lng
+
+_Data Format_: all creators should live in a creators.yml file, similar to sites.yml.
+
+**Jobs** have the following fields:
+* Title
+* Location
+* Organization (FK)
+* (Optional) salary low end
+* (Optional) salary high end
+* (Optional) salary currency
+* Contact email or application link. 
+* Excerpt (short version)
+* Description
+
+_Data Format_: Jobs should be stored in markdown files in a /jobs folder. Jobs should be stored in markdown files. Frontmatter should have all fields except description. Body of the file should be the description. 
+
+### Notes on long-term data strategy
+* Right now, the data is in YAML in the GitHub repo. At some point, we may move all of our data to a format with a UI making it easier for people to submit & edit their own data, without imposing a review burden on Gatsby maintainers
+* Option 1: use Airtable or a CMS
+    * Airtable doesn’t have an easy way to allow users to edit individual forms in a record. 
+    * We could build something on top with Lambdas, or utilize the Zapier => Airtable integration: https://support.airtable.com/hc/en-us/articles/115007655568-Using-forms-to-edit-tables-with-Zapier
+    * We could also make job postings go down after 60 days
+* Option 2: Build an application ourselves
+    * Could do this, but what’s the value here?
+    * We probably don’t want to spin up an application to do this
+    * Firebase as backend?
+* **Option 3: Stick with YAML in Github for now (what we’re doing)**
+    * We know this works, although it does add additional maintainer burden reviewing these files
+    * If we find a good global “application” type solution, we probably want to migrate the sites in the site showcase here as well. 
+    * **Almost certainly the right move for now.**
```

Comment:
I think it's fine for a proof of concept, but I think manually managed relational databases are going to become a nightmare pretty much instantly after launch.